mne/io/eeglab/eeglab.py

```diff
@@ -92,8 +109,10 @@ def _get_info(eeg, montage, eog=()):
                 if not np.any(np.isnan(locs)):
                     pos_ch_names.append(chanloc.labels)
                     pos.append(locs)
+        if err_ch_set != set():
+            print('\n')
```

Comment:
Sure thing. I'm puzzled though, why does `
`if chanloc.type in _kind_dict:`
generate 
`TypeError: unhashable type: 'numpy.ndarray'` ?
When is chanloc.type not a string?